UnitTests/PayPalTests/MockCheckout.swift

```diff
@@ -3,49 +3,34 @@ import UIKit
 
 @testable import PayPalNativeCheckout
 @testable import PaymentsCore
+import PayPalCheckout
+import XCTest
 
 class MockCheckout: CheckoutProtocol {
 
-    static var onApprove: ApprovalCallback?
-    static var onCancel: CancelCallback?
-    static var onError: ErrorCallback?
+    required init(nxoConfig: CheckoutConfig) {
+    }
 
-    static func set(config: CoreConfig, clientID: String) { }
+    var onCancel: CheckoutConfig.CancelCallback?
+    var onError: CheckoutConfig.ErrorCallback?
 
-    static func start(
+    // todo: implemenet cases for other callbacks
+    // swiftlint:disable function_parameter_count
```

Comment:
Instead of silencing the linter for these violations, we should probably update our linter to suit our needs. It [warns at 5 params by default](https://realm.github.io/SwiftLint/function_parameter_count.html) which is low, especially for an SDK whose main job is to wrap lengthy/complex API calls.

Reply:
I agree 5 is too low, may be 7? ignored this because it's in tests not part of sdk

Reply:
We could exclude this rule entirely too, adding it [here](https://github.com/paypal/iOS-SDK/blob/295dbaaa0f0e98bb0bd60d2e721bf15262d8afbf/.swiftlint.yml#L11-L18).

Due to the nature of us being an SDK that does a lot of wrapping API requests and results, I don't think the limit on function param count really helps us that much. What do you think?

Reply:
Agree with Sammy, we should probably just disable this rule since we are likely to run into this often with building SDKs